regress/unittests/test_helper/test_helper.c

```diff
@@ -133,6 +133,28 @@ main(int argc, char **argv)
 	ERR_load_CRYPTO_strings();
 #endif
 
+#ifdef WINDOWS
```

Comment:
With the below code, you run unittest with moduli file in %programdata%\\ssh. Without this change, you run unitest with moduli file in __progdir directory. Anyway, can only cover one scenario. Which one is the most common scenario?

Reply:
Having it in %programdata%\ssh is the better.